.travis.yml

```diff
@@ -27,7 +27,7 @@ deploy:
   provider: script
   script: heroku container:release web -a devops-trg
   on: 
-    branch: master
+    branch: mod-10-oauth
```

Comment:
Nice - I can see that this deployed well and is probably working, although it errors as I don't have permissions (that's fine for here). Don't forget to switch this back on merge :)